tests/django_hstore_tests/tests/test_schema_mode.py

```diff
@@ -65,6 +66,17 @@ def test_dict_get(self):
         self.assertEqual(d.get('default_test', 'default'), 'default')
         self.assertIsNone(d.get('default_test'))
 
+    def test_save_and_reload_custom_type(self):
+        d = SchemaDataBag()
+        v = CustomType('some value')
+        d.custom = v
+        self.assertEqual(d.custom, v)
+
+    def test_serializing_custom_type(self):
+        d = SchemaDataBag()
+        d.custom = CustomType('some value')
+        self.assertEqual(dict.get(d.data, 'custom'), '[some value]')
```

Comment:
If the first test passes even after saving/reloading the instance, then this one is not really needed. `dict.get(d.data, 'custom')` is pretty much what `d.data.get('custom')` calls when calling `super().get()`.

When writing this test I assumed the serialization would be done when assigning values, and only later did I realize that's not how native Django fields work.